app/models/concerns/orchestration/tftp.rb

```diff
@@ -40,6 +40,9 @@ def generate_pxe_template
     # therefore some workaround is required to "render" the template.
     @kernel = host.operatingsystem.kernel(host.arch)
     @initrd = host.operatingsystem.initrd(host.arch)
+    if host.operatingsystem.pxe_type == 'yast'
+      @mediapath = host.operatingsystem.mediumpath(host)
+    end
```

Comment:
Looks like the `mediumpath` is defined for all OSes except Gentoo (returns `nil`). Can we simplify it and set it for all OSes (fixing Gentoo to return an empty string rather than nil)? Also an extra test for Red Hat would be nice.